pom.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.hotels</groupId>
+    <artifactId>hotels-oss-parent</artifactId>
+    <version>2.0.6</version>
+  </parent>
+
+  <groupId>com.hotels</groupId>
+  <artifactId>hcommon-hive-metastore</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <inceptionYear>2017</inceptionYear>
+
+  <properties>
+    <hadoop.version>2.7.1</hadoop.version>
+    <hive.version>2.3.2</hive.version>
+    <log4j.version>1.2.17</log4j.version>
+    <spring-platform.version>Brussels-SR5</spring-platform.version>
+    <mockito.version>1.10.19</mockito.version>
+    <hamcrest-library.version>1.3</hamcrest-library.version>
+    <beeju.version>1.2.1</beeju.version>
+  </properties>
+
+  <dependencyManagement>
```

Comment:
do we need Spring in this library?

Reply:
I just use one of the SpringTest libraries in a test, I will try and find the specific dependency, we don't need the platform-bom

Reply:
it doesn't look like we need spring at all, in which test you need it? We should try to get rid of this